crypto/cipher_extra/aead_test.cc

```diff
@@ -54,8 +56,7 @@ constexpr uint32_t RequiresADLength(size_t length) {
   // if (length >= 16) {
   //  __builtin_unreachable();
   // }
-  return (length & 0xf) << 3;
-}
+  return (length & 0xf) << 3;}
```

Comment:
- `assert.h` is #included. Is it used? Same question about `type_check.h` which is not in the upstream commit
- Do we want the closing brace to be on the same line as the `return` statement?

Reply:
Good catch, that was a hold over from my attempt to reconcile the merge conflict using `OPENSSL_STATIC_ASSERT`.